website/content/docs/agent-and-proxy/proxy/caching/index.mdx

```diff
@@ -36,6 +37,21 @@ specific scenarios.
    that are issued using the tokens managed by the proxy, will be cached and
    its renewals are taken care of.
 
+## Static secret caching
+
+Vault Proxy can be configured to cache KVv1 and KVv2 secrets as well as dynamic secrets described
+above. This means that, when enabled, subsequent requests to the same KV secret made via Proxy will
+only result in a single request made to Vault. Proxy will keep a single cache entry of the secret in its
+cache, and will only provide the cached response to requests made with tokens that can access the secret.
+
+This functionality off by default, as it requires [auto-auth](/vault/docs/agent-and-proxy/autoauth) to be configured,
+as well as the auto-auth token to have the permissions to subscribe to KV event updates via Vault's [event system](/vault/docs/concepts/events).
+Proxy uses the auto-auth token to connect to Vault's event system, and uses events to know when to update its cache
+with updated secrets.
```

Comment:
```suggestion
Static secret caching is disabled by default. To enable secret caching you must
configure [auto-auth](/vault/docs/agent-and-proxy/autoauth) and ensure the
auto-auth token has permission to subscribe to KV
[event](/vault/docs/concepts/events) updates.

Once configured, Proxy uses the auto-auth token to subscribe to KV events, and
monitors the subscription feed to know when to update the secrets in its cache.
```

Style correction: write in active voice, use declarative statements, avoid "this" as a pronoun

Reply:
Thank you! I'll update this :)